test/e2e/proxy_test.go

```diff
@@ -692,7 +693,7 @@ func TestSpaceLister(t *testing.T) {
 
 			// then
 			require.NoError(t, err)
-			verifyHasExpectedWorkspace(t, expectedWorkspaceFor(t, awaitilities.Host(), users["car"], true), *workspace)
+			verifyHasExpectedWorkspace(t, expectedWorkspaceFor(t, awaitilities.Host(), users["car"], true, appStudioTierRolesWSoption), *workspace)
```

Comment:
Is it enough to add the roles to the expected WS? Do we actually check anywhere that the expected roles present?

Reply:
From what I can see, by adding the roles to the expected WS the `verifyHasExpectedWorkspaceFor` function will compare the `actualWorkspace.Status` with the `expectedWorkspace.Status` here: https://github.com/codeready-toolchain/toolchain-e2e/blob/6135f39caa4a45a677cc46854e0a7efb0e502d49/test/e2e/proxy_test.go#L1047

so it should be fine, but unfortunately I didn't get to run this test till the end yet because of the other failure I've reported in the slack thread.

Reply:
Yes, you are right. I missed the status check.